src/World.h

```diff
@@ -1051,13 +1049,13 @@ class cWorld :
 	cWorld(const AString & a_WorldName, eDimension a_Dimension = dimOverworld, const AString & a_OverworldName = "");
 	virtual ~cWorld();
 
-	void Tick(float a_Dt, int a_LastTickDurationMSec);
+	void Tick(std::chrono::milliseconds a_Dt, std::chrono::milliseconds a_LastTickDurationMSec);
```

Comment:
a_Dt is how long between the start of the last tick and this on, a_LastTickDurationMSec is how long the last tick took to execute. I'm not sure anything uses a_LastTickDurationMSec.